moped-editor/src/views/projects/newProjectView/NewProjectView.js

```diff
@@ -492,14 +432,21 @@ const NewProjectView = () => {
     // eslint-disable-next-line
   }, [saveActionState]);
 
+  if (componentQueryloading) {
+    return <CircularProgress />;
+  }
+  if (componentQueryError) return `Error! ${componentQueryError.message}`;
```

Comment:
Did not put any thought to loading state and error state renders—just copied what i saw in ProjectSummary view.